EduPerfTests/Performance.cs

```diff
@@ -87,34 +64,35 @@ public void JetStream(Browser browser, RemoteWebDriver driver, int iterations)
                     Thread.Sleep(10000);
                 }
                 var result = driver.FindElementById("results-cell-geomean").Text.Substring(0, 6);
-                logFile.WriteLine(string.Format("JetStream,{0},{1},{2},", browser, result, i + 1));
+                _perfLog.WriteToLog($"JetStream,{browser},{result},{i + 1},");
             }
         }
 
         public void WebXPRT(Browser browser, RemoteWebDriver driver, int iterations)
         {
-            for (int i = 0; i < iterations; i++)
+            for (var i = 0; i < iterations; i++)
             {
                 driver.Url = "http://www.principledtechnologies.com/benchmarkxprt/webxprt/2015/v19982/";
                 driver.FindElementById("imgRunAll").Click();
-                while (driver.Url != "http://www.principledtechnologies.com/benchmarkxprt/webxprt/2015/v19982/results.php?c=0")
+                while (driver.Url !=
+                        "http://www.principledtechnologies.com/benchmarkxprt/webxprt/2015/v19982/results.php?c=0")
                 {
                     Thread.Sleep(10000);
                 }
                 var result = driver.FindElementByCssSelector(".resultsOval>.scoreText").Text;
-                logFile.WriteLine(string.Format("WebXPRT,{0},{1},{2},", browser, result, i + 1));
+                _perfLog.WriteToLog($"WebXPRT,{browser},{result},{i + 1},");
             }
         }
 
         public void OORTOnline(Browser browser, RemoteWebDriver driver, int iterations)
```

Comment:
I know you didn't change this method (other than some ReSharper refactoring which is always nice. I'd use it if I had a license for it.). We need to probably revisit this performance test (maybe not in this PR unless you want to try to take a whack at it). It doesn't write anything out to the Document but instead is all rendered. That's why I have a huge wait (used IE as my long pole on a terrible machine to get the wait time) and I take a picture instead. If we value this test we might want to try to find a better way to more accurately determine when it's complete and snag a screenshot.